pkg/kubelet/status/manager_test.go

```diff
@@ -64,7 +65,8 @@ func (m *manager) testSyncBatch() {
 
 func newTestManager(kubeClient client.Interface) *manager {
 	podManager := kubepod.NewBasicPodManager(kubepod.NewFakeMirrorClient())
-	podManager.AddPod(testPod)
```

Comment:
Ha...I saw your respond, the page is not refreshed just now. If so I'll send a following PR to refactor the test a little.